app/views/users/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<table>
+  <tr>
+    <th>
+      Username:
+    </th>
+    <th>
+      Password:
+    </th>
+  </tr>
+<% @users.each do |user| %>
```

Comment:
Excellent use of looping here! 

Be careful of your indentation. This loop and its contents should be 2 spaces indented from the `<table>` tag